includes/main.php

```diff
@@ -131,6 +131,8 @@ public function load()
 		// Functions shared in both back end and front end.
 		include_once 'functions/shared.php';
 
+		// Oauth ajax action
+		include_once 'admin/auth-service-actions.php';
```

Comment:
May I know why position for this file include is changed?

Reply:
Previously I we use code only for admin and now I had added event display code in this file. So now we need this file on both the front end backend.